app/src/androidTest/java/fr/free/nrw/commons/MainActivityTest.kt

```diff
@@ -0,0 +1,37 @@
+package fr.free.nrw.commons
+
+import android.content.res.Configuration
+import androidx.test.filters.MediumTest
+import androidx.test.rule.ActivityTestRule
+import androidx.test.runner.AndroidJUnit4
+import fr.free.nrw.commons.contributions.MainActivity
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import android.R.attr.orientation
+
+
+
+@MediumTest
+@RunWith(AndroidJUnit4::class)
+class MainActivityTest {
+    @get:Rule
+    var activityRule = ActivityTestRule(MainActivity::class.java)
+
+    @Test
+    fun orientationChange(){
+        val firstConfiguration = Configuration()
+        val nextConfiguration = getOtherConfiguration()
+        activityRule.activity.onConfigurationChanged(nextConfiguration)
+        assert(firstConfiguration != nextConfiguration)
+    }
+
+    private fun getOtherConfiguration(): Configuration {
```

Comment:
This can be moved to `UITesthelper` so that other tests can reuse it.

Reply:
Also maybe rename to `getOtherRotationConfiguration` so it's more clear what configuration it is referring to exactly.

Reply:
I'll do that :+1: 
Apart from that, is everything correct? Shall I start working on other activities in the same PR (or may be different for some)?

Reply:
I'm just going to test - will have result in 5 mins.

Reply:
Also, I think doing a sample one and asking for feedback was a great idea btw!

Reply:
Thanks @domdomegg :)